setup.cfg

```diff
@@ -79,3 +79,35 @@ source =
 
 [tool:pytest]
 testpaths = tests
+
+[mypy]
+files = src/flask_caching/
+warn_unused_configs = true
+
+# TODO if all of these can be enabled we can just set 'strict = true'
+disallow_subclassing_any = true
+# disallow_untyped_calls = true
+# disallow_untyped_defs = true
+# disallow_incomplete_defs = true
+# check_untyped_defs = true
```

Comment:
This should probably have the highest priority to enable, since it will highlight existing type error.

<details>

```
src/flask_caching/utils.py:94: error: Argument 1 to "translate" of "str" has incompatible type "*Tuple[Dict[str, None]]"; expected "Union[Mapping[int, Union[int, str, None]], Sequence[Union[int, str, None]]]"
src/flask_caching/utils.py:97: error: Argument 1 to "translate" of "str" has incompatible type "*Tuple[Dict[str, None]]"; expected "Union[Mapping[int, Union[int, str, None]], Sequence[Union[int, str, None]]]"
src/flask_caching/backends/rediscache.py:94: error: "str" not callable
src/flask_caching/backends/rediscache.py:111: error: Incompatible types in assignment (expression has type "List[Any]", variable has type "Tuple[Any, ...]")
src/flask_caching/__init__.py:409: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "cache_timeout"
src/flask_caching/__init__.py:499: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "uncached"
src/flask_caching/__init__.py:500: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "cache_timeout"
src/flask_caching/__init__.py:501: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "make_cache_key"
src/flask_caching/__init__.py:599: error: Argument "timeout" to "_memoize_version" of "Cache" has incompatible type "Union[Any, Callable[..., Any], None]"; expected "Optional[int]"
src/flask_caching/__init__.py:838: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "make_cache_key"
src/flask_caching/__init__.py:885: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "cache_timeout"
src/flask_caching/__init__.py:893: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "uncached"
src/flask_caching/__init__.py:894: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "cache_timeout"
src/flask_caching/__init__.py:895: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "make_cache_key"
src/flask_caching/__init__.py:898: error: Argument "forced_update" to "_memoize_make_cache_key" of "Cache" has incompatible type "Optional[Callable[..., Any]]"; expected "bool"
src/flask_caching/__init__.py:903: error: "Callable[[VarArg(Any), KwArg(Any)], Any]" has no attribute "delete_memoized"
Found 16 errors in 3 files (checked 14 source files)
```

</details>